frontend/Dockerfile

```diff
@@ -1,11 +1,15 @@
 # Base image for the container
-FROM node:20.13.1
+FROM node:20.14.0
+
+ENV npm_config_cache=/app/.npm
+ENV APP_USER=1011540000
 
 # Copy files, install dependencies and build
+USER ${APP_USER}
 WORKDIR /app/
-COPY . ./
+COPY --chown=${APP_USER} . ./
```

Comment:
Good to know about the `--chown` flag for the `COPY` here :smile:

Reply:
It's great! Thanks for the comments. :D